docs/user-guide/getting-started.md

```diff
@@ -11,175 +11,173 @@ The Ambassador Edge Stack is designed to run in Kubernetes for production. The m
 
 * Kubernetes 1.11 or later
 * The `kubectl` command-line tool
+* Edge Control
 
-## Install the Ambassador Edge Stack
+## Quick Install (Recommended!)
 
 The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) image to deploy the Ambassador Edge Stack locally.
 
-1. In your terminal, run the following command:
+When you use Edge Control on your publicly
+accessible cluster, it will:
 
-    ```bash
-    kubectl apply -f https://www.getambassador.io/yaml/aes-crds.yaml && \
-    kubectl wait --for condition=established --timeout=90s crd -lproduct=aes && \
-    kubectl apply -f https://www.getambassador.io/yaml/aes.yaml && \
-    kubectl -n ambassador wait --for condition=available --timeout=90s deploy -lproduct=aes
-    ```
-
-2. Determine the IP address of your cluster by running the following command:
-
-    ```bash
-    kubectl get -n ambassador service ambassador -o 'go-template={{range .status.loadBalancer.ingress}}{{print .ip "\n"}}{{end}}'
-    ```
+1. Install the Ambassador Edge Stack
+2. Generate a domain name for you to access the Edge Policy Console and complete
+   advanced configuration
+3. Obtain a TLS certificate for that domain name
+4. Configure automatic TLS and HTTPS using that certificate
 
-    Your load balancer may take several minutes to provision your IP address. Repeat the provided command until you get an IP address.
+**To install the Ambassador Edge Stack:**
```

Comment:
What's your thought on starting the section with an infinitive? (You don't do this anywhere else?)

Reply:
I actually try to start any steps with either "To do so" or "To do <action>" and then the steps.